src/components/navigation/NavItems.js

```diff
@@ -70,11 +73,34 @@ const PortalOuterWrapper = styled.div`
 const PortalInnerWrapper = styled.div`
   background-color: white;
   box-shadow: 0 5px 20px rgba(0, 0, 0, 0.05);
-  min-width: ${(props) => (props.portal === PRODUCTS ? '300px' : '200px')};
+  min-width:
+    ${(props) => {
+    if (props.portal === PRODUCTS) {
+      return '300px';
+    }
+    if (props.portal === RESOURCES) {
+      return '200px';
+    }
+    return '200px';
```

Comment:
This logic is identical to what was there before, i.e. it'll only return `300px` for `PRODUCTS` and `200px` otherwise.

Reply:
oops you're totally right